scripts/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
-cargo build --release --target wasm32-unknown-unknown -p mitex-typst
-rm -f packages/mitex/mitex.wasm
-mv target/wasm32-unknown-unknown/release/mitex_typst.wasm packages/mitex/mitex.wasm
+cargo build --release --target wasm32-unknown-unknown --manifest-path ./crates/mitex-wasm/Cargo.toml --features typst-plugin
+cp target/wasm32-unknown-unknown/release/mitex_wasm.wasm packages/mitex/mitex.wasm
+
+cd crates/mitex-wasm
+wasm-pack build --release --features web
```

Comment:
Some developers may have not installed wasm-pack. So I think there is a good point to separate a build-web.sh. But I also don't like continuing to increase the number of scripts.

@Enter-tainer and @OrangeX4

Reply:
> Some developers may have not installed wasm-pack

maybe just install them.